pkg/shares/shares_test.go

```diff
@@ -176,19 +176,24 @@ func Test_zeroPadIfNecessary(t *testing.T) {
 		width int
 	}
 	tests := []struct {
```

Comment:
[for reviewers] I found it a little confusing that the `zeroPadIfNecessary` function is defined in `parse_sparse_shares.go` but tested in `shares_test.go` when I would've expected it to be tested in `parse_sparse_shares_test.go`.

Additionally this function is used by sparse shares so proposal to move the function to `utils.go` (and this test to `utils_test.go`)

will do this as a follow-up if reviewers agree

Reply:
yeah why not. my guess was this was just an oversight when moving things around the past few weeks